### PR TITLE
Upgrading vulnerable packages

### PR DESCRIPTION
**Pull request by jmprieur**
# Upgrading vulnerable packages

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -355,3 +355,5 @@ MigrationBackup/
 /tests/E2E Tests/PlaywrightTraces
 /tests/IntegrationTests/PlaywrightTraces
 /.SharedData
+/out/
```
Comment (jmprieur) [inline .gitignore:358]: these folders are created by newest versions of msbuild (when ran on the command line)

**Review by jennyf19 — APPROVED** [review]
:shipit:

File: tests/PerformanceTests/PerformanceTestService/PerformanceTestService.csproj [hunk 2]
```diff
@@ -24,7 +24,7 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.ApplicationInsights.AspNetCore" Version="$(MicrosoftApplicationInsightsAspNetCoreVersion)" />
-    <PackageReference Include="Microsoft.ApplicationInsights.EventCounterCollector" Version="$(MicrosoftApplicationInsightsEventCounterCollectionVersion)" />
```
Comment (westin-m) [inline tests/PerformanceTests/PerformanceTestService/PerformanceTestService.csproj:27]: Why replace this instead of updating the value of this variable?

Comment (jmprieur) [reply tests/PerformanceTests/PerformanceTestService/PerformanceTestService.csproj:27]: good catch. I let VS do.

**Review by westin-m — APPROVED** [review]

File: tests/PerformanceTests/PerformanceTestService/PerformanceTestService.csproj (hunk 2 above)
Comment (jmprieur) [reply tests/PerformanceTests/PerformanceTestService/PerformanceTestService.csproj:27]: done

**Pull request merged**